### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by loa**
- Added documentation to readme
- Collapsed all examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,135 @@
...
+
+## What is this?
+
+This is our current try to create standards for building, testing and deploying our applications.
+
+What we want:
+- General pipelines that can build, test and deploy all our apps/services
+- Useful development tooling for local development
+
+Putting the logic in the app repos would create a lot of duplications, big overhead when creating new services and maintenance hell. Putting the logic in the pipelines will make local development difficult since all logic for building/testing/deploying exists in the CI systems. This implies that logic for building/testing/deploying apps/services need to be accessible for both developers and CI system.
```
Comment (stanislavb) [inline README.md]: "creating new services and maintenance hell" could be reshuffled to not imply that the user gets overhead when creating maintenance hell

"logic [...] needs* to be accessible"

File: README.md [hunk 2]
```diff
@@ -1,3 +1,135 @@
...
+
+This is our current try to create standards for building, testing and deploying our applications.
+
+What we want:
+- General pipelines that can build, test and deploy all our apps/services
+- Useful development tooling for local development
+
+Putting the logic in the app repos would create a lot of duplications, big overhead when creating new services and maintenance hell. Putting the logic in the pipelines will make local development difficult since all logic for building/testing/deploying exists in the CI systems. This implies that logic for building/testing/deploying apps/services need to be accessible for both developers and CI system. 
+
+Golang was choosen for Wrench since we can have automatic cross-compiled binaries for linux and osx.
```
Comment (stanislavb) [inline README.md]: "chosen"

Capitalize Linux and OSX

File: README.md [hunk 3]
```diff
@@ -1,3 +1,135 @@
...
+
+Wrench is provided as [docker image](https://registry.hub.docker.com/u/tomologic/wrench/) and through our [homebrew tap](https://github.com/tomologic/homebrew-tap) for osx users.
```
Comment (stanislavb) [inline README.md]: OSX

File: README.md [hunk 4]
```diff
@@ -1,3 +1,135 @@
...
+  organization: example
+  name: real-app-name
+  version: v1.0.0
+```
+
+## Build
+
+There are 2 ways that wrench can build docker images. Simple mode which is the normal docker approach and the builder which is uses a separate image to build the application artifact _(very useful for golang)_.
+
+The mode is choosen depending on the existing Dockerfiles in current directory. Builder mode will override simple though, which makes it possible to use [Automated Builds on Docker Hub](https://docs.docker.com/docker-hub/builds/) with the regular Dockerfile and the builder image for usage with wrench.
```
Comment (stanislavb) [inline README.md]: chosen

**Comment by stanislavb** [comment]
Super awesome documentation. Moar grammar and spelling (as commented) and it's perfect :dart:

File: README.md (hunk 1 above)
Comment (loa) [inline README.md]: Fixed!

File: README.md (hunk 2 above)
Comment (loa) [inline README.md]: Fixed!

File: README.md (hunk 3 above)
Comment (loa) [inline README.md]: Fixed!

File: README.md (hunk 4 above)
Comment (loa) [inline README.md]: Fixed!

**Pull request merged**